src/es/latanime/src/eu/kanade/tachiyomi/animeextension/es/latanime/Latanime.kt

```diff
@@ -284,11 +284,11 @@ class Latanime : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
             val prefix = "${videoElement.text()} - "
 
             when {
-                url.contains("ok.ru") -> {
+                url.contains("ok") -> {
```

Comment:
isn't there a better way to match this link pattern?
any significantly long path could contain the letters "ok"